src/cascadia/CascadiaResources.build.items

```diff
@@ -2,247 +2,331 @@
     </Content>
     <Content Include="$(MSBuildThisFileDirectory)CascadiaPackage\ProfileIcons\{0caa0dad-35be-5f56-a8ff-afceeeaa6101}.scale-200.png">
+      <DeploymentContent>true</DeploymentContent>
       <Link>ProfileIcons\{0caa0dad-35be-5f56-a8ff-afceeeaa6101}.scale-200.png</Link>
     </Content>
     <Content Include="$(MSBuildThisFileDirectory)CascadiaPackage\ProfileIcons\{574e775e-4f2a-5b96-ac1e-a2962a402336}.scale-100.png">
+      <DeploymentContent>true</DeploymentContent>
       <Link>ProfileIcons\{574e775e-4f2a-5b96-ac1e-a2962a402336}.scale-100.png</Link>
     </Content>
     <Content Include="$(MSBuildThisFileDirectory)CascadiaPackage\ProfileIcons\{574e775e-4f2a-5b96-ac1e-a2962a402336}.scale-200.png">
+      <DeploymentContent>true</DeploymentContent>
       <Link>ProfileIcons\{574e775e-4f2a-5b96-ac1e-a2962a402336}.scale-200.png</Link>
     </Content>
     <Content Include="$(MSBuildThisFileDirectory)CascadiaPackage\ProfileIcons\{61c54bbd-c2c6-5271-96e7-009a87ff44bf}.scale-100.png">
+      <DeploymentContent>true</DeploymentContent>
       <Link>ProfileIcons\{61c54bbd-c2c6-5271-96e7-009a87ff44bf}.scale-100.png</Link>
     </Content>
     <Content Include="$(MSBuildThisFileDirectory)CascadiaPackage\ProfileIcons\{61c54bbd-c2c6-5271-96e7-009a87ff44bf}.scale-200.png">
+      <DeploymentContent>true</DeploymentContent>
       <Link>ProfileIcons\{61c54bbd-c2c6-5271-96e7-009a87ff44bf}.scale-200.png</Link>
     </Content>
     <Content Include="$(MSBuildThisFileDirectory)CascadiaPackage\ProfileIcons\{9acb9455-ca41-5af7-950f-6bca1bc9722f}.scale-100.png">
+      <DeploymentContent>true</DeploymentContent>
       <Link>ProfileIcons\{9acb9455-ca41-5af7-950f-6bca1bc9722f}.scale-100.png</Link>
     </Content>
     <Content Include="$(MSBuildThisFileDirectory)CascadiaPackage\ProfileIcons\{9acb9455-ca41-5af7-950f-6bca1bc9722f}.scale-200.png">
+      <DeploymentContent>true</DeploymentContent>
       <Link>ProfileIcons\{9acb9455-ca41-5af7-950f-6bca1bc9722f}.scale-200.png</Link>
     </Content>
     <Content Include="$(MSBuildThisFileDirectory)CascadiaPackage\ProfileIcons\{b453ae62-4e3d-5e58-b989-0a998ec441b8}.scale-100.png">
+      <DeploymentContent>true</DeploymentContent>
       <Link>ProfileIcons\{b453ae62-4e3d-5e58-b989-0a998ec441b8}.scale-100.png</Link>
     </Content>
     <Content Include="$(MSBuildThisFileDirectory)CascadiaPackage\ProfileIcons\{b453ae62-4e3d-5e58-b989-0a998ec441b8}.scale-200.png">
+      <DeploymentContent>true</DeploymentContent>
       <Link>ProfileIcons\{b453ae62-4e3d-5e58-b989-0a998ec441b8}.scale-200.png</Link>
     </Content>
     <!-- Default Settings -->
     <Content Include="$(MSBuildThisFileDirectory)\TerminalApp\defaults.json">
+      <DeploymentContent>true</DeploymentContent>
       <Link>defaults.json</Link>
     </Content>
   </ItemGroup>
+  <!-- Reprint all content as Nones for Universal project type to include it in the resources -->

```

Comment:
@DHowett-MSFT, this reprint as None and the DeploymentContent on every item are killing me, but it's the only way I could make it work. If you know some MSBuild shorthand to clean this up, we can do that before submit (as long as both projects still play nice with it.)